### PR TITLE
8130 creating a new company in twenty doesnt activate on zapier

### LINKED ISSUE #8130: Creating a new company in twenty doesn't activate on Zapier
I have setup a zap that should trigger immediately when a customer is created. I have manually added a customer, and added through another API however it doesn't seem to trigger. When setting up the zap it connects perfectly and finds previous entries and when combining it with the second zap it sends those entries off however this is only when running it as a test. 

Could this be a bug or a known issue or something on my end that has been misconfigured.

Self hosted instance on docker. Replication steps:
1) Connect to zapier and setup a trigger when a company is created
2) Connect a second app (google contacts etc)
3) Link the fields to creating a new contact. You will see already created companies appear as expected. Save the zap
4) Add new company in twenty (through adding the details onto a new line in the companies, or adding a new company through the API). Save
5) No task is created in zapier, and a new contact doesn't get created.

Screenshot 2024-10-28 at 10 21 23 am
Screenshot 2024-10-28 at 10 22 21 am

### PR DESCRIPTION
**Pull request by martmull**
- fix webhook.operation format change from august 2024 not spread in twenty-zapier
- added a comment so it does not happen again
- add a fix for the new webhook.operations column that would produce another issue

### CODE REVIEW

File: packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook-jobs.job.ts [hunk 1]
```diff
@@ -42,6 +42,9 @@ export class CallWebhookJobsJob {
 
   @Process(CallWebhookJobsJob.name)
   async handle(data: CallWebhookJobsJobData): Promise<void> {
+    // If you change that function, double check it does not break Zapier
```
Comment (charlesBochet) [inline packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook-jobs.job.ts:45]: ok for now, I think we should share types between server and zapier (likely a package twenty-shared with ts interfaces later)

Comment (martmull) [reply packages/twenty-server/src/engine/api/graphql/workspace-query-runner/jobs/call-webhook-jobs.job.ts:45]: Would love such a package

File: packages/twenty-server/src/database/commands/upgrade-version/0-32/0-32-copy-webhook-operation-into-operations-command.ts [hunk 2]
```diff
@@ -23,6 +24,30 @@ export class CopyWebhookOperationIntoOperationsCommand extends ActiveWorkspacesC
     super(workspaceRepository);
   }
 
+  private async reverseOperationIfNecessary(
```
Comment (charlesBochet) [inline packages/twenty-server/src/database/commands/upgrade-version/0-32/0-32-copy-webhook-operation-into-operations-command.ts:27]: I don't see this code being used

File: packages/twenty-server/src/database/commands/upgrade-version/0-32/0-32-copy-webhook-operation-into-operations-command.ts [hunk 3]
```diff
@@ -23,6 +24,30 @@ export class CopyWebhookOperationIntoOperationsCommand extends ActiveWorkspacesC
...
+    const [firstWebhookPart, lastWebhookPart] = webhook.operation.split('.');
+
+    if (['created', 'updated', 'deleted'].includes(firstWebhookPart)) {
+      await webhookRepository.update(webhook.id, {
+        operation: `${lastWebhookPart}.${firstWebhookPart}`,
+      });
+    }
+  }
+
+  private async copyOperationToOperations(
```
Comment (charlesBochet) [inline packages/twenty-server/src/database/commands/upgrade-version/0-32/0-32-copy-webhook-operation-into-operations-command.ts:42]: same here

**Review by charlesBochet — APPROVED** [review]
Approved but left comments :)

**Pull request merged**